src/Bridges/Variable/map.jl

```diff
@@ -572,18 +572,34 @@ function function_for(map::Map, ci::MOI.ConstraintIndex{MOI.VectorOfVariables})
     return MOI.VectorOfVariables(variables)
 end
 
+"""
+    struct CannotUnbridgeVariableFunction(msg::String) <: Exception
+
+The error thrown when Variable bridges do not support unmapping the function.
+
+This is usually because the forward mapping is lossy, for example, when the
+`ZerosBridge` converts `x in Zeros()` to `0.0`; given the constant `0.0`, we
+cannot recover the variable `x`.
+"""
+struct CannotUnbridgeVariableFunction <: Exception
```

Comment:
Another approach would be to define an attribute `UnbridgedVariableFunction <: MOI.AbstractVariableAttribute` and to throw here `GetAttributeNotAllowed{UnbridgedVariableFunction}` so that we don't have to try catch and change it to a different attribute. It's not obeying the MethodError principle but I don't think that's the only place we have that ^^ Otherwise we'll have to `try`-`catch` every single line of code

Reply:
The alternative is just to throw `GetAttributeNotAllowed{MOI.ConstraintFunction}`, which is pretty much what we do below anyway.